src/Data/Nat/Binary/Base.agda

```diff
@@ -106,6 +113,38 @@ _        * zero     =  zero
 1+[2 x ] * 2[1+ y ] =  2[1+ y + x * 2[1+ y ] ]
 1+[2 x ] * 1+[2 y ] =  1+[2 x + y * 1+[2 x ] ]
 
+_-_ : ℕᵇ → ℕᵇ → Maybe ℕᵇ
```

Comment:
There's some inconsistency here between the names. We have the `ᴹ` in `predᴹ` signifying that it returns a `Maybe` but `_-_` returns a `Maybe` but has no `ᴹ`. To be honest I think the `Maybe` versions should be the canonical ones (i.e. have the simple name) and the non-`Maybe` versions should have a derived name.

What are other people's thoughts?

Reply:
I actually agree with you on that the `Maybe` versions should be the canonical ones, but if we rename `predᴹ` into `pred` and the original `pred` into something else, we would break the convention followed by `Data.Nat.Base` as there `pred` is defined without `Maybe`.

Reply:
That's a good point. Okay, let's leave it lie for now.